### PR TITLE
PSR-565 Held any shares page

### PR DESCRIPTION
**Pull request by kberec**

### CODE REVIEW

File: app/pages/nonsipp/shares/DidSchemeHoldAnySharesPage.scala [hunk 1]
```diff
@@ -14,16 +14,15 @@
...
+package pages.nonsipp.shares
 
 import models.SchemeId.Srn
 import pages.QuestionPage
 import play.api.libs.json.JsPath
 
-case class UnquotedSharesPage(srn: Srn) extends QuestionPage[Boolean] {
+case class DidSchemeHoldAnySharesPage(srn: Srn) extends QuestionPage[Boolean] {
 
   override def path: JsPath = JsPath \ toString
```
Comment (cyeniceri) [inline app/pages/nonsipp/shares/DidSchemeHoldAnySharesPage.scala:25]: Shouldn't we have any parent JsPath here for shares ?

File: app/controllers/nonsipp/sharesacquiredfromconnectedparty/SharesAcquiredFromConnectedPartyController.scala [hunk 2]
```diff
@@ -1,108 +0,0 @@
-/*
- * Copyright 2023 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
```
[the remark this answers is not in the document]
Comment (kberec) [reply app/controllers/nonsipp/sharesacquiredfromconnectedparty/SharesAcquiredFromConnectedPartyController.scala:4]: They aren't needed anymore.

File: app/navigation/nonsipp/SharesNavigator.scala [hunk 3]
```diff
@@ -16,22 +16,18 @@
...
 
   override def normalRoutes: UserAnswers => PartialFunction[Page, Call] = userAnswers => {
 
-    case page @ DidSchemeHoldSharesInSponsoringEmployerPage(srn) =>
-      if (userAnswers.get(page).contains(true)) {
-        controllers.nonsipp.unquotedshares.routes.UnquotedSharesController.onPageLoad(srn, NormalMode)
-      } else {
-        controllers.nonsipp.routes.TaskListController.onPageLoad(srn)
-      }
+    case page @ DidSchemeHoldAnySharesPage(srn) =>
```
[the remark this answers is not in the document]
Comment (kberec) [reply app/navigation/nonsipp/SharesNavigator.scala:29]: Yes, but it's going to change in the very next PR: https://github.com/hmrc/pension-scheme-return-frontend/pull/443/files

File: app/pages/nonsipp/shares/DidSchemeHoldAnySharesPage.scala (hunk 1 above)
Comment (kberec) [reply app/pages/nonsipp/shares/DidSchemeHoldAnySharesPage.scala:25]: Good spot, if you agree, I'll add them in with the next data item (ticket I'll be doing next)

Comment (cyeniceri) [reply app/pages/nonsipp/shares/DidSchemeHoldAnySharesPage.scala:25]: I think that will be good to see the json structural when we want to troubleshoot any issue. I think we are doing this in other pages if we don't miss any.

**Review by cyeniceri — APPROVED** [review]

File: app/pages/nonsipp/shares/DidSchemeHoldAnySharesPage.scala (hunk 1 above)
Comment (kberec) [reply app/pages/nonsipp/shares/DidSchemeHoldAnySharesPage.scala:25]: Yes, we need that. I made a note and address it with the next ticket that has relevant data: [PSR-568]

**Pull request merged**